docs/docs/providers/shopify.md

```diff
@@ -0,0 +1,45 @@
+---
+id: shopify
+title: Shopify
+---
+
+## Documentation
+
+https://shopify.dev/apps/auth/oauth/getting-started
+
+## Configuration
+
+This is your **Redirect URL** <ins>http://localhost:3000/api/auth/callback/shopify</ins>. Add this callback URL in your app settings.
```

Comment:
I would re-word this. 

`http://localhost:3000/api/auth/callback` works if the user is trying the application locally, however in other environments like production the domain is going to be different. 

For instance, when using **Vercel** it would be: `https://my-app.vercel.app/api/auth/callback`.

```suggestion
Add `/api/auth/callback/shopify in your app settings as the **callback URL**. For instance, if you're trying the sign in locally it'll be: `http://localhost:3000/api/auth/callback`.
```